packages/wallet/scripts/start.js

```diff
@@ -56,7 +56,7 @@ void (async () => {
   }
 
   // Tools like Cloud9 rely on this.
-  const DEFAULT_PORT = !!process.env.PORT ? parseInt(process.env.PORT, 10) : 3000;
+  const DEFAULT_PORT = !!process.env.PORT ? parseInt(process.env.PORT, 10) : 3055;
```

Comment:
`ganache-deployer` starts up a web server on port 3000, so this was changed to avoid collisions.